api/client/service.proto

```diff
@@ -37,10 +39,40 @@ message ExecutionResponse {
   }
 }
 
+message RequestStreamRequest {
+  uint64 amount = 1;
```

Comment:
I'm curious why the client needs to specify this at all? Is this some form of basic flow control so that the client isn't overwhelmed?

Reply:
Yes, added a comment about that. 
I couldn't find a way to cleanly add this capability otherwise (without that becoming a significant effort in itself).